docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,209 @@
+| `frontend.namespaceCount`              | Int  | Rate limit on concurrent Task Queue polls per Namespace per instance.                                                                                                                                                              | 1200          |
+| `frontend.globalNamespaceRPS`          | Int  | Namespace rate limit per second applied globally on the entire Cluster. The limit is evenly distributed among available Frontend service instances. If this is set, it overrides the per-instance limit (`frontend.namespaceRPS`). | 0             |
+| `internal-frontend.globalNamespaceRPS` | Int  | Namespace rate limit per second across all internal-frontends.                                                                                                                                                                     | 0             |
+| History                                |      |                                                                                                                                                                                                                                    |               |
+| `history.rps`                          | Int  | Request rate per second for each History host.                                                                                                                                                                                     | 3000          |
+| Matching                               |      |                                                                                                                                                                                                                                    |               |
+| `matching.rps`                         | Int  | Request rate per second for each Matching host                                                                                                                                                                                     | 1200          |
+| `matching.numTaskqueueReadPartitions`  | Int  | Number of read partitions for a Task Queue. Must be set with `matching.numTaskqueueWritePartitions`.                                                                                                                               | 4             |
+| `matching.numTaskqueueWritePartitions` | Int  | Number of write partitions for a Task Queue.                                                                                                                                                                                       | 4             |
+
+### QPS limits for Persistence database
```

Comment:
Global question: Instead of  "Persistence database" and "Persistence store database," should we use "Persistence store" consistently throughout?

Reply:
Yep; changed to Persistence store across.